index.js

```diff
@@ -147,11 +150,17 @@ class SlackNotifier extends NotificationBase {
             buildData.event.commit.message;
         const isMinimized = buildData.settings.slack.minimized;
 
+        let notificationStatus = buildData.status;
+
+        if (buildData.status === 'SUCCESS' && buildData.isFixed) {
+            notificationStatus = 'FIXED';
+        }
+
         let message = isMinimized ?
             // eslint-disable-next-line max-len
             `<${pipelineLink}|${buildData.pipeline.scmRepo.name}#${buildData.jobName}> *${buildData.status}*` :
             // eslint-disable-next-line max-len
-            `*${buildData.status}* ${STATUSES_MAP[buildData.status]} <${pipelineLink}|${buildData.pipeline.scmRepo.name} ${buildData.jobName}>`;
+            `*${notificationStatus}* ${STATUSES_MAP[buildData.status]} <${pipelineLink}|${buildData.pipeline.scmRepo.name} ${buildData.jobName}>`;
```

Comment:
Does this need to be set to notificationStatus as well? Or we can remove the new entry on line 33
```suggestion
            `*${notificationStatus}* ${STATUSES_MAP[notificationStatus]} <${pipelineLink}|${buildData.pipeline.scmRepo.name} ${buildData.jobName}>`;
```

Reply:
I fixed. Thank you review. 
The reason I'm setting the value as a variable is because
If you are using more than one notification-plugin, it uses the same `buildData.status.`

Changing  `buildData.status = 'FIXED' ` directly will break it.